config.json

```diff
@@ -34,6 +34,41 @@
   },
   "exercises": {
     "concept": [],
+    "foregone-comments": [
+      "Some of the reasons to forego exercises:",
+      "- specific data structures (trees, linked lists, etc)",
+      "    - these are _possible_ but not practical",
+      "- very object-oriented",
+      "- threading/concurrency",
+      "- refactoring exercises (un-forego if you're willing to write them)",
+      "See https://github.com/exercism/awk/issues/105"
+    ],
```

Comment:
We don't yet support formatting the `config.json` file, but I'm not sure whether we'll support custom properties. No reason to do anything about it now, but just wanted to mention it.

Reply:
Noted.